recipes/magic_wormhole/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "magic-wormhole" %}
+{% set version = "0.10.5" %}
+{% set sha256 = "" %}
```

Comment:
:scream:

Reply:
I forgot about that, I'm glad that I'm being a good human fuzzer. Something tells me that this isn't supported behavior?

Reply:
We should lint for that. I guess we even have an open issue for it :smile: